brightray/browser/inspectable_web_contents_impl.h

```diff
@@ -92,7 +92,7 @@ class InspectableWebContentsImpl
   void AppendToFile(const std::string& url,
                     const std::string& content) override;
   void RequestFileSystems() override;
-  void AddFileSystem(const std::string& file_system_path) override;
+  void AddFileSystem(const std::string& type) override;
```

Comment:
the path is no longer needed...?

Reply:
The path was always empty when invoked from devtools, users get to pick the folder with the file dialog the client provides and we get the path from it. The type of filesystem was added in https://chromium-review.googlesource.com/c/chromium/src/+/729250, have made the changes to reflect it. Also verified it works as expected.